app/controllers/obituaries/community_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Obituaries::CommunityController < ApplicationController
+  def index
+    @obituaries = Obituary.all
```

Comment:
This might have to change a bit once we figure out how api links and our database obits are intermingled.

Reply:
for sure. are we ordering these in any particular way or do we want to?